cmd.js

```diff
@@ -3,6 +3,7 @@
 
 var chokidar = require('chokidar')
 var child = require('child_process')
+const objectAssign = require('object-assign');
```

Comment:
`const` isn't supported in node 0.12 and the semicolon is against standard code style